sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,13 @@
+
+//Colors
+$col-white:#fff;
+$col-black:#000;
+$col-header:#ff0000;
+$col-primary:#41aae6;
+$col-primary-bg:#41aae6d1;

```

Comment:
Colours should be hex values, either 3 or 6 characters after a hash. https://www.pluralsight.com/blog/tutorials/understanding-hexadecimal-colors-simple